### PR TITLE
fix: websocket test failed

### PR DESCRIPTION
**Pull request by lsytj0413**
go/websocket test failed, run command:
`
cd websocket
go test -v ./...
`

go error:
`=== RUN   TestWSClient
--- PASS: TestWSClient (1.00s)
=== RUN   TestWSPing
10
--- PASS: TestWSPing (2.00s)
=== RUN   TestWSServer
--- FAIL: TestWSServer (1.00s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x6689fc]

goroutine 34 [running]:
testing.tRunner.func1(0xc42016c000)
        /usr/lib/go-1.9/src/testing/testing.go:711 +0x2d2
panic(0x6c11a0, 0x8a6740)
        /usr/lib/go-1.9/src/runtime/panic.go:491 +0x283
github.com/gorilla/websocket.(*Conn).SetPongHandler(0x0, 0x72e318)
        /home/soren/workspace/golang/lib/src/github.com/gorilla/websocket/conn.go:1116 +0x2c
github.com/lsytj0413/go/websocket.TestWSServer(0xc42016c000)
        /home/soren/workspace/golang/src/github.com/lsytj0413/go/websocket/server_test.go:68 +0x1e9
testing.tRunner(0xc42016c000, 0x72e320)
        /usr/lib/go-1.9/src/testing/testing.go:746 +0xd0
created by testing.(*T).Run
        /usr/lib/go-1.9/src/testing/testing.go:789 +0x2de
exit status 2
FAIL    github.com/lsytj0413/go/websocket       4.121s`

### CODE REVIEW

File: websocket/client_test.go [hunk 1]
```diff
@@ -1,67 +1,80 @@
...
 	"github.com/gorilla/websocket"
 )
 
 func TestWSClient(t *testing.T) {
+	var wg sync.WaitGroup
+	wg.Add(1)
+
 	http.HandleFunc("/test/client", func(w http.ResponseWriter, r *http.Request) {
+		defer func() {
+			fmt.Println("wg.Done")
```
Comment (siddontang) [inline websocket/client_test.go]: it is not a good style to use fmt and log in the test directly, you can use t.Log or t.Logf instead.

File: websocket/client_test.go [hunk 2]
```diff
@@ -1,67 +1,80 @@
...
-
+		if msgType != websocket.TextMessage {
 			t.Fatal("invalid msg type", msgType)
 		}
+		log.Println("server conn.ReadMessage")
+		conn.WriteMessage(websocket.PongMessage, []byte("server.Pong"))
 	})
 
-	go http.ListenAndServe(":65500", nil)
+	go http.ListenAndServe(":65501", nil)
```
Comment (siddontang) [inline websocket/client_test.go]: why changing the port?

Comment (siddontang) [inline websocket/client_test.go]: why changing the port?

File: websocket/client_test.go (hunk 1 above)
Comment (lsytj0413) [reply websocket/client_test.go]: 3ks, i will fix it.

**Pull request merged**